### PR TITLE
Policies can be linked to people

### PR DESCRIPTION
**Pull request by tekin**
Allows linking of policy areas and policy programmes to people. Build will be broken until https://github.com/alphagov/govuk-content-schemas/pull/30 is merged.

Trello: https://trello.com/c/cZ7P1VfR/51-add-ability-to-tag-policies-to-orgs-and-people

### CODE REVIEW

**Comment by tekin** [comment]
Re-built now that content schema has been merged.

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 
 $LOAD_PATH << File.join(File.dirname(__FILE__), '..')
+# require 'rspec/rails'
```
Comment (elliotcm) [inline spec/spec_helper.rb]: Youse forgots to removes dis

Comment (tekin) [inline spec/spec_helper.rb]: Removed!

File: features/step_definitions/policy_area_steps.rb [hunk 2]
```diff
@@ -46,8 +50,28 @@
...
+      "publishing_app" => "policy-publisher",
+      "locale" => "en",
+      "links" => {
+        "organisations" => [],
+        "people" => [person_1["content_id"]],
+        "related" => [],
+      },
+    }
+  )
+end
```
Comment (elliotcm) [inline features/step_definitions/policy_area_steps.rb]: Whitespace.

File: spec/presenters/content_item_presenter_spec.rb [hunk 3]
```diff
@@ -39,5 +39,13 @@
 
       expect(attributes["links"]["organisations"]).to eq([content_id])
     end
+
+    it "includes linked people with a policy area" do
+      content_id = SecureRandom.uuid
+      policy_area = FactoryGirl.create(:policy_area, people_content_ids: [content_id])
+      attributes = ContentItemPresenter.new(policy_area).exportable_attributes.as_json
```
Comment (elliotcm) [inline spec/presenters/content_item_presenter_spec.rb]: Could you switch this to `to_json`?  Part of my work is changing some of these specs to not require Rails, and `as_json` is a Rails addon.

**Comment by elliotcm** [comment]
Lots of duplication again, though we should address that properly under separate cover.

**Comment by elliotcm** [comment]
I've spoken to @edds about the duplication (in this app in general), he's going to arrange something to go over it properly.

**Pull request merged**